### ISSUE TITLE
Rename the project

### ISSUE THREAD

**Issue by Crandel** [nuIIpointerexception/vterm#4]
### Check for existing issues

- [X] Completed

### Describe the feature

There are already terminal emulator with the same name
https://github.com/man9ourah/vterm
Please rename your project in order to avoid confusion.

### If applicable, add some sort of preview to help us visualize your idea.

https://github.com/man9ourah/vterm

**Comment by Riptils** [comment]
Some people have thought of Vulterm on reddit as a new nickname.
I suggest using **Wulter**
The W stands for Wayland protocol
The (V)ul for vulkan
and ter for terminal.
Thanks again

**Comment by nuIIpointerexception** [comment]
Good suggestion, i personally don't think that vterm is a very good sounding name. So i'm open to any suggestions.

Hmm, since i plan on supporting all major platforms i don't really see wayland anywhere in the name, but i already thought of something like **volterm** or just **volt** in short. That would be available on the AUR too, what do you think?

**Comment by Riptils** [comment]
> Bonne suggestion, personnellement, je ne pense pas que vterm soit un très bon nom. Je suis donc ouvert à toutes suggestions.
> 
> Hmm, puisque je prévois de prendre en charge toutes les principales plates-formes, je ne vois vraiment Wayland nulle part dans le nom, mais j'ai déjà pensé à quelque chose comme ** volterm ** ou simplement ** volt ** en bref. Cela serait également disponible sur l'AUR, qu'en pensez-vous ?

I find that the name "volt" is a bit impersonal, and there are already projects in the AUR with similar names like volta, voltron, voltdb, revolt, volti... Personally, I don't really understand the interest in a terminal on Windows; the audience is really small... On macOS, I think the people who want to tinker with experimental terminals are also very few. I think it would be better to focus on the Linux audience, who use the command line every day. In this sense, I thought it would be good to integrate a bit of Wayland into the name because it is an essential part of input/display handling, and the combination of Vulkan, Rust, and Wayland can somewhat present this project as future-proof. I hope you understand where I am coming from. Thank you for the development.

PS: I have a request for the development. Could you avoid making a terminal with lots of features/aesthetic stuff and gadgets, and rather focus on something minimal, lightweight, and very performant (above all)? I switched from Alacritty to Foot for this reason, and having a terminal as heavy as Alacritty or even Kitty is not what I'm looking for with this terminal. I hope you will take my suggestions into account.

**Comment by nuIIpointerexception** [comment]
> > Bonne suggestion, personnellement, je ne pense pas que vterm soit un très bon nom. Je suis donc ouvert à toutes suggestions.
> > Hmm, puisque je prévois de prendre en charge toutes les principales plates-formes, je ne vois vraiment Wayland nulle part dans le nom, mais j'ai déjà pensé à quelque chose comme ** volterm ** ou simplement ** volt ** en bref. Cela serait également disponible sur l'AUR, qu'en pensez-vous ?
> 
> I find that the name "volt" is a bit impersonal, and there are already projects in the AUR with similar names like volta, voltron, voltdb, revolt, volti... Personally, I don't really understand the interest in a terminal on Windows; the audience is really small... On macOS, I think the people who want to tinker with experimental terminals are also very few. I think it would be better to focus on the Linux audience, who use the command line every day. In this sense, I thought it would be good to integrate a bit of Wayland into the name because it is an essential part of input/display handling, and the combination of Vulkan, Rust, and Wayland can somewhat present this project as future-proof. I hope you understand where I am coming from. Thank you for the development.
> 
> PS: I have a request for the development. Could you avoid making a terminal with lots of features/aesthetic stuff and gadgets, and rather focus on something minimal, lightweight, and very performant (above all)? I switched from Alacritty to Foot for this reason, and having a terminal as heavy as Alacritty or even Kitty is not what I'm looking for with this terminal. I hope you will take my suggestions into account.

Yeah i understand what you mean, i haven't really written any goals down inside the repo anyway, so thanks for the suggestion!

Short info to the project:
I'm looking to create something very performant but also very customizable.
My goal is to create a project that can be extended using extensions, but also the ability to have a very lightweight experience.

So basically the first install will be as barebones as possible, while people can still decide if they want some sort of AI Copilot or whatever inside their terminal. Basically a neovim but for terminals powered by vulkan.

So yeah, ill try my best keeping everything compact and easy.

Just out of curiousity: what about kitty do you not like in terms of heavy-being?

**Comment by Riptils** [comment]
> Just out of curiousity: what about kitty do you not like in terms of heavy-being?

I don't like it because it's heavy, kitty can easily go up to over 100mb of ram whereas foot is usually around 10-15mb and alacritty also uses around a hundred mb. so I prefer foot because it's much, much more minimalist. And I'm waiting for a terminal of the same type for this project.
